js/components/graph/Container.js

```diff
@@ -9,8 +9,8 @@ export default class Container extends React.Component {
   render() {
     return (
       <React.Fragment>
-        <Graph start_blank={this.props.start_blank} edit={this.props.edit} initialDrawMode="draw-node" />
-        <Sidebar />
+        <Graph start_blank={this.props.start_blank} edit={this.props.edit} initialDrawMode="draw-node" ref={instance => {this.graph = instance; }}/>
```

Comment:
Rather than creating a local ref on the Graph in the render method, try creating a ref in the `Container` constructor so now we can use that ref in all of our methods in this file. Otherwise, any method outside of the render won't be able to reference this ref. You can see `Graph.js` for details.